courses/machine_learning/deepdive2/supplemental/solutions/object_detection_with_automl.ipynb

```diff
@@ -0,0 +1,559 @@
+{
```

Comment:
<p>Explain what you mean by "annotations" and explain directly here what you think needs to be explained instead of pointing to the documentation. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/1065/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>

Reply:
<p>Wanted to state here that image labeling management can be done only in the UI of AutoML Vision after the dataset has already been created. Can remove this</p>